### PR TITLE
add absinthe engine

### PR DESCRIPTION
**Pull request by cataliniovita**
Add a new entry for Absinthe engine: https://github.com/absinthe-graphql/absinthe
Test: https://52.27.153.235/
         https://147.139.171.54/

### CODE REVIEW

File: graphw00f/helpers.py [hunk 1]
```diff
@@ -255,7 +255,13 @@ def get_engines():
       'url':'https://github.com/appointy/jaal',
       'ref':'https://github.com/nicholasaleks/graphql-threat-matrix/blob/master/implementations/jaal',
       'technology':['Golang']
-    }
+    },
+    'absinthe-graphql':{
+      'name':'absinthe-graphql',
+      'url':'https://github.com/absinthe-graphql/absinthe',
+      'ref':'https://github.com/nicholasaleks/graphql-threat-matrix/blob/master/implementations/absinthe-graphql.md',
```
Comment (dolevf) [inline graphw00f/helpers.py:262]: Ideally this should exist as well (separate PR) so enumeration of absinthe can also detail what security features it has or doesn't have

Comment (cataliniovita) [reply graphw00f/helpers.py:262]: @dolevf So should I create a separate PR with these modifications?

Comment (dolevf) [reply graphw00f/helpers.py:262]: yep, please

Comment (cataliniovita) [reply graphw00f/helpers.py]: Done, please take a look: https://github.com/dolevf/graphw00f/pull/31

**Pull request merged**